dev/com.ibm.websphere.appserver.features/visibility/public/mpTelemetry-1.0/io.openliberty.mpTelemetry-1.0.feature

```diff
@@ -12,6 +12,7 @@ IBM-API-Package: \
   io.opentelemetry.api.trace;type="third-party",\
   io.opentelemetry.api.common;type="third-party",\
   io.opentelemetry.context;type="third-party",\
+  io.opentelemetry.sdk.trace;type="third-party",\
```

Comment:
Why does this package need to be added?

Reply:
This is not required. It's been taken out now

Reply:
I got this error: 
```
2022-12-12-13:17:53:741 ERROR: Caught exception attempting to call test method callMethodAnnotatedWithParameter on servlet io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.WithSpanServlet java.lang.NoClassDefFoundError: io.opentelemetry.sdk.trace.ReadableSpan at io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.WithSpanServlet$SpanBean.methodAnnotatedWithParameter(WithSpanServlet.java:111) at io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.WithSpanServlet$SpanBean$Proxy$_$$_WeldSubclass.methodAnnotatedWithParameter$$super(Unknown Source) at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) at java.base/java.lang.reflect.Method.invoke(Method.java:566) at org.jboss.weld.interceptor.proxy.TerminalAroundInvokeInvocationContext.proceedInternal(TerminalAroundInvokeInvocationContext.java:51) at org.jboss.weld.interceptor.proxy.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:78) at io.openliberty.microprofile.telemetry.internal.cdi.WithSpanInterceptor.span(WithSpanInterceptor.java:92) at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) at java.base/java.lang.reflect.Method.invoke(Method.java:566) at
```

Reply:
Sure, but you can't just expose packages in the API because it makes testing easier, you need to justify why the package needs to be accessible by applications and usually the list of accessible packages is laid down in the design document.

In this case I think we do need it anyway since the autoconfigure SPI references `io.opentelemetry.sdk.trace.SdkTracerProviderBuilder` [here][reference], but otherwise we'd have to find a different way to do this test.

[reference]: https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/1.19.0/io/opentelemetry/sdk/autoconfigure/spi/AutoConfigurationCustomizer.html#addTracerProviderCustomizer(java.util.function.BiFunction)